### PR TITLE
[Internal] Refactored fix builds

### PR DESCRIPTION
**Pull request by optout21**
The fix builds, and all the Typescript scripts used by them have been refactored completely.
Most TS script has been moved/duplicated.

### CODE REVIEW

File: script/action/binance.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import * as path from "path";
+import * as chalk from 'chalk';
+
+import {
+    getChainAssetLogoPath,
+    getChainBlacklistPath
+} from "../common/repo-structure";
+
+const binanceChain = "binance"
+const binanceAssetsUrl = "https://explorer.binance.org/api/v1/assets?page=1&rows=1000";
```
Comment (vikmeup) [inline script/action/binance.ts:13]: add to config

File: script/action/update-all.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export function fixAll() {
+    console.log("Running fixes...");
+    eth_forks.fix();
+    logo_size.fix();
+    validators.fix();
+    whitelists.fix();
+}
+
+export function updateAll() {
+    console.log("Running updates (using external data sources) ...");
```
Comment (vikmeup) [inline script/action/update-all.ts:18]: can you delete tezos from this API? all of the related code? besides JSON structure - keep it as is

File: script/common/image.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import {
+    writeFileSync,
+    getFileSizeInKilobyte
+} from "./filesystem";
+import * as chalk from 'chalk';
+
+//export const minLogoWidth = 64;
+//export const minLogoHeight = 64;
+export const maxLogoWidth = 512;
+export const maxLogoHeight = 512;
```
Comment (vikmeup) [inline script/common/image.ts:13]: add to global config

File: script/action/binance.ts (hunk 1 above)
Comment (catenocrypt) [reply script/action/binance.ts]: done

File: script/action/update-all.ts (hunk 2 above)
Comment (catenocrypt) [reply script/action/update-all.ts:18]: Let's keep Tezos validator remval in separate PR

File: script/common/image.ts (hunk 3 above)
Comment (catenocrypt) [reply script/common/image.ts]: done

File: script/config.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+    "binanceAssetsUrl": "https://explorer.binance.org/api/v1/assets?page=1&rows=1000"
```
Comment (vikmeup) [inline script/config.json]: `const binanceAssetsUrl = config.getConfig("binanceAssetsUrl", "https://explorer.binance.org/api/v1/assets?page=1&rows=1000");`

this does not make sense

Comment (vikmeup) [reply script/config.json]: This should be statically typed

**Pull request merged**